cl/settings/project/security.py

```diff
@@ -9,6 +9,10 @@
     "ALLOWED_HOSTS", default=["www.courtlistener.com"]
 )
 
+WEBHOOKS_PROXY_HOST = env(
+    "WEBHOOKS_PROXY_HOST", default="http://cl-webhook-sentry:9090"
+)
```

Comment:
I think in the future there's a good chance we'll start using this for other things too, like scraping. Better to call it `EGRESS_PROXY_HOST` instead?

Reply:
Changed!